make doc fails when building with chapel 2.5.0

When building arkouda with chapel 2.5.0, make doc fails with the following error:
```
rm -f docs/server/index.html
Completed building documentation for: Python
find . -name "*.pyi" -type f -delete
Building documentation for: Server
chpldoc --process-used-modules /home/amandapotts/git/arkouda/src/registry/doc-support.chpl /home/amandapotts/git/arkouda/src/arkouda_server.chpl /home/amandapotts/git/arkouda/src/compat/eq-22/* -o /home/amandapotts/git/arkouda/docs/server
warning in /home/amandapotts/git/arkouda/src/compat/eq-22/ArkoudaSparseMatrixCompat.chpl:2: 'LayoutCS' and its 'CS' layout are deprecated; please use 'CompressedSparseLayout' and its 'csrLayout' or 'cscLayout' layouts instead
warning in /home/amandapotts/git/arkouda/src/GenSymIO.chpl:33: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/GenSymIO.chpl:53: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/GenSymIO.chpl:82: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/GenSymIO.chpl:123: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/AryUtil.chpl:1004: Unknown attribute tool name 'arkouda'
error in /home/amandapotts/git/arkouda/src/compat/eq-22/ArkoudaSortCompat.chpl:2: cannot use 'except' clause with symbol 'DefaultComparator' as it is not defined in 'Sort'
warning in /home/amandapotts/git/arkouda/src/MsgProcessing.chpl:34: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/MsgProcessing.chpl:48: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/MsgProcessing.chpl:313: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/MsgProcessing.chpl:328: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/MsgProcessing.chpl:357: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/BigIntMsg.chpl:19: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/BigIntMsg.chpl:61: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/BigIntMsg.chpl:98: Unknown attribute tool name 'arkouda'
warning in /home/amandapotts/git/arkouda/src/BigIntMsg.chpl:113: Unknown attribute tool name 'arkouda'
make: *** [Makefile:653: /home/amandapotts/git/arkouda/docs/server/index.html] Error 1

```

--- a/src/compat/eq-22/ArkoudaSortCompat.chpl
+++ b/src/compat/eq-22/ArkoudaSortCompat.chpl
@@ -1,5 +1,5 @@
 module ArkoudaSortCompat {
-  public use Sort except defaultComparator, DefaultComparator;
+  public use Sort except defaultComparator;
 
   proc defaultComparator type {
     import Sort;

